ai-mocks-core/src/commonMain/kotlin/me/kpavlov/aimocks/core/json/schema/SchemaHelper.kt

```
@@ -0,0 +1,115 @@
+
+    /**
+     * Parses a [JsonElement] into a [SchemaDefinition].
+     *
+     * @param schemaJson The JSON element containing the schema
+     * @return The parsed schema definition, or null if parsing fails
+     */
+    public fun parseSchema(schemaJson: JsonElement?): SchemaDefinition? =
+        schemaJson?.let {
+            try {
+                json.decodeFromJsonElement(SchemaDefinition.serializer(), it)
+            } catch (e: Exception) {
```

detekt (1.23.8)

[warning] The caught exception is too generic. Prefer catching specific exceptions to the case that is currently handled.

(detekt.exceptions.TooGenericExceptionCaught)